features/admin.applications.v1/pages/applications.tsx

```diff
@@ -443,43 +445,45 @@ const ApplicationsPage: FunctionComponent<ApplicationsPageInterface> = (
                                 floated="left"
                                 mobile={ 16 }
                                 computer={ 9 }
+                                verticalAlign="middle"
                             >
                                 <GenericIcon
                                     icon={ getGeneralIcons().myAccountSolidIcon }
-                                    className="mt-1"
                                     floated="left"
-                                    size="tiny"
+                                    size="mini"
                                     spaced="right"
                                     verticalAlign="middle"
                                     inline
                                     square
                                     transparent
                                 />
-                                <List.Header
-                                    data-componentid="application-consumer-account-link-title"
-                                    className="my-account-title mb-1"
-                                >
-                                    { t("applications:myaccount.title") }
-                                    {
-                                        applicationConfig?.advancedConfigurations?.showMyAccountStatus && (
-                                            <Icon
-                                                color={ isMyAccountEnabled ? "green":"grey" }
-                                                name={ isMyAccountEnabled ? "check circle" : "minus circle" }
-                                                className="middle aligned ml-1"
-                                            />
-                                        )
-                                    }
-                                </List.Header>
-                                <List.Description
-                                    data-componentid="application-consumer-account-link-description"
-                                >
-                                    { t("applications:myaccount.description") }
-                                    <DocumentationLink
-                                        link={ getLink("develop.applications.myaccount.learnMore") }
+                                <List.Content verticalAlign="middle">
+                                    <List.Header
+                                        data-componentid="application-consumer-account-link-title"
+                                        className="my-account-title"
+                                    >
+                                        { t("applications:myaccount.title") }
+                                        {
+                                            applicationConfig?.advancedConfigurations?.showMyAccountStatus && (
```

Comment:
I think we don't need this config anymore. 

cc: @JayaShakthi97

Reply:
will track this in a later PR